src/sass/components/_products.scss

```diff
@@ -0,0 +1,93 @@
+@use '../utils/forward' as *;
+
+.products {
+    min-width: 320px;
+    max-width: 768px;
```

Comment:
zamień to na 1200px;
i dodaj selector:
`&__container: {
  max-width: 768px;
  margin: 0 auto;
  padding: 121px 20px 98px;
}`